### DISCUSSION TITLE
Hosts same subnet cannot access SOC

### DISCUSSION THREAD

**Discussion by baylorex97** [Security-Onion-Solutions/securityonion#6171] [Unsupported Versions] [answered]
This is probably not a SO issue, but hoping maybe someone has run into this.

SO latest ISO installed on standalone Linux server.  All of the hosts are VM's running on different servers.  SO works great. Port mirrors gathering traffic and reporting on different networks.  No issues until I tried loading OSQuery on two hosts, one Linux, one Windows.  OSQuery fails to connect.  Error on both hosts are https://x.x.x.x:8090, i/o timeout.  Trying to connect to correct mgmt ip

Troubleshooting:
Both hosts are on the same subnet as the Mgmt SO interface
Hosts can ping SO Mgmt ip
Hosts cannot connect to SOC ip (Mgmt int, configure as ip only, no fqdn).  Had to scp the OSQuery download files over.
The OSQuery flag files have the correct ip.
Hosts on other subnets can access the SOC
Traffic does not hit the SO server.  No errors in logs, no iptable blocks, no pcap packets from hosts.
So-allow is configured for 0.0.0.0/0 for all modes (analyst, osquery, wazuh, etc)
All servers are connected to same switch.
SO Mgmt port is configured as access port, access VLAN xxx.
Created a new Linux VM, installed same SO ISO , same config (different mgmt ip) on it.  No access issues.  Hosts on same subnet can access the console, OSQuery on hosts show up in Fleet.

I'm lost, hoping someone has some answers.

**Comment by dougburks** [comment, marked as answer]
> This is probably not a SO issue

Yeah, this doesn't sound like a Security Onion issue.

> SO Mgmt port is configured as access port, access VLAN xxx.

You might try double-checking the VLAN settings and making sure there is nothing else that could be interfering with the traffic.

**Comment by baylorex97** [comment]
That's what I thought.  Thanks for the reply.
